doc/user_guide/psy_data.rst

```diff
@@ -146,24 +151,29 @@ printed before and after each kernel call that is checked. If the
 variable is set to '2', it will additionally print the name of each
 variable that is checked.
 
-
 Read-Only Verification Library for LFRic
 ++++++++++++++++++++++++++++++++++++++++
+
 This library is contained in ``lib/read_only/lfric`` and it must be compiled
 before compiling any LFRic-based application that uses read-only verification.
 Compiling this library requires access to the LFRic infrastructure library
-(since it must implement a generic interface for e.g. the LFRic ``Field`` class).
-The makefile uses the variable ``LFRIC_DIR`` to point to the location where
-LFRic's ``field_mod`` has been compiled. It defaults to
-``../../../../../lfric/work/trunk/miniapps/gravity_wave/working/field``
-but this will certainly need to be changed for any user. The LFRic
-infrastructure library is not used in linking the verification library.
-The application which uses the read-only-verification library needs to
-link in the infrastructure library anyway.
+(since it must implement a generic interface for e.g. the LFRic
+:ref:`field <lfric-field>` class).
+
+The ``Makefile`` uses the variable ``INF_DIR`` to point to the location
```

Comment:
I was about to say it's not INFDIR, but it is now - I admit I was wondering about the different environment variables in the past, and thought that it would be nice to use the same ... and it looks like you did it, great!

Reply:
Yep, I was confused why there were two variables for infrastructure so I decided to make it more consistent.